client/application.js

```diff
@@ -940,23 +836,19 @@ export default {
                   }
                 ]
               },
-              {
-                name: 'quantity',
-                label: 'How many of these animals will be used in this protocol?',
-                type: 'text'
-              },
               {
                 name: 'continued-use',
-                label: 'Have any of these animals had procedures applied to them in preparation for their use in this protocol (continued use)?',
+                label: 'Will any animals coming on to this protocol be classed as ‘continued use’?',
+                hint: '‘Continued use’ describes animals that have had procedures applied to them in order to be used in a particular protocol. For example, genetically altered animals that have been bred for scientific use.',
                 type: 'radio',
                 options: [
                   {
                     label: 'Yes',
                     value: true,
                     reveal: {
-                      name: 'continued-use-sourced',
-                      label: 'Where will these animals be sourced from?',
-                      hint: 'This could be another protocol in this licence, another project licence, or somewhere else.',
+                      name: 'continued-use-start',
```

Comment:
Can we keep this field name consistent?

Reply:
ooh good spot - I thought I caught them all